packages/jsii-pacmak/test/generated-code/examples.test.ts

```diff
@@ -0,0 +1,67 @@
+import { SPEC_FILE_NAME } from '@jsii/spec';
+import * as fs from 'fs-extra';
+import * as jsii from 'jsii';
+import * as os from 'os';
+import * as path from 'path';
+
+import * as pacmak from '../../lib';
+import { checkTree, TREE } from './harness';
+
+const EXAMPLES_ROOT = path.resolve(__dirname, 'examples');
+
+for (const name of fs.readdirSync(EXAMPLES_ROOT)) {
+  const file = path.join(EXAMPLES_ROOT, name);
+  test(name, async () => {
+    const source = await fs.readFile(file, 'utf8');
+    const compiled = await jsii.compileJsiiForTest(source);
+
+    const tmpdir = await fs.mkdtemp(os.tmpdir());
+    try {
+      // Create pretend source tree
+      await fs.writeJson(path.join(tmpdir, SPEC_FILE_NAME), compiled.assembly, {
+        encoding: 'utf8',
+        spaces: 2,
+      });
+      await fs.writeJson(path.join(tmpdir, 'package.json'), {
+        name: compiled.assembly.name,
+        version: compiled.assembly.version,
+        jsii: {
+          outdir: 'dist',
+          targets: {
+            go: { moduleName: 'example.test/demo' },
+          },
```

Comment:
why not include all the target types already?

Reply:
Speed of iteration 🙃